Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -671,6 +677,12 @@ protected bool IsTextSearchable
         protected string DropdownClassNames
             => $"{Class} b-is-autocomplete {( Multiple ? "b-is-autocomplete-multipleselection" : string.Empty )} {( TextFocused ? "focus" : string.Empty )}";
 
+        /// <summary>
+        /// Gets the custom class-names for dropdown element.
+        /// </summary>
+        protected string DropdownItemClassNames(int index)
+            => $"b-auto-complete-suggestion {(ActiveItemIndex == index ? "focus" : string.Empty)}";
```

Comment:
Naming should be consistent

![image](https://user-images.githubusercontent.com/900302/172305728-57fba69d-86de-4746-8ec5-0803c0b36cf1.png)

Reply:
It's strange that we used `b-is-{component}` instead of `b-{component}` tough. Might be an inconsistency to fix. 
May I add to our notes that the naming for components should be `b-{component}` and we fix any inconsitency like this in 2.0?

For example:
![image](https://user-images.githubusercontent.com/22283161/172439373-9c97f38e-da9f-416c-bb9e-8b1836b99aa2.png)